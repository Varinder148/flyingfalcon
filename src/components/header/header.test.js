import { shallow } from "enzyme";
import Header from "./header.component";

describe("Header tests", () => {
  it("checks if Header is correct", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
