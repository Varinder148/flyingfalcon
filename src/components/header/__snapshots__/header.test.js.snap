// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests checks if Header is correct 1`] = `ShallowWrapper {}`;
